pilot/pkg/model/context.go

```diff
@@ -394,6 +394,8 @@ const (
 
 	// IstioIngressNamespace is the namespace where Istio ingress controller is deployed
 	IstioIngressNamespace = "istio-system"
+
+	IstioInboundCaptureAllPort = "INBOUND_CAPTURE_ALL_PORT"
```

Comment:
This is the wrong place for this metadata.. please list it with all other node metadata. And also, this is the wrong name format.

Reply:
And probably we should pass the value of includeInboundPorts - instead of helm-script to convert to bool (and lose info).

Can also be used to improve other things.

Reply:
@rshriram Thank you for the patience for the repeat. I previously commented that I didn't find a existing example...  Could you elaborate?